lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -30,66 +28,121 @@ class MinitestAssertions < Base
           RESTRICT_ON_SEND = %i[
             assert_equal
             assert_not_equal
-            refute_equal
             assert_nil
             assert_not_nil
+            assert_empty
+            assert_not_empty
+            refute_equal
             refute_nil
+            refute_empty
           ].freeze
 
-          # @!method minitest_equal_assertion(node)
-          def_node_matcher :minitest_equal_assertion, <<~PATTERN
+          # @!method minitest_equal(node)
+          def_node_matcher :minitest_equal, <<~PATTERN
             (send nil? {:assert_equal :assert_not_equal :refute_equal} $_ $_ $_?)
           PATTERN
 
-          # @!method minitest_nil_assertion(node)
-          def_node_matcher :minitest_nil_assertion, <<~PATTERN
+          # @!method minitest_nil(node)
+          def_node_matcher :minitest_nil, <<~PATTERN
             (send nil? {:assert_nil :assert_not_nil :refute_nil} $_ $_?)
           PATTERN
 
+          # @!method minitest_empty(node)
+          def_node_matcher :minitest_empty, <<~PATTERN
+            (send nil? {:assert_empty :assert_not_empty :refute_empty} $_ $_?)
+          PATTERN
+
           def on_send(node)
-            minitest_equal_assertion(node) do |expected, actual, fail_message|
-              prefer = replace_equal_assertion(node, expected, actual,
-                                               fail_message.first)
-              add_an_offense(node, prefer)
+            minitest_equal(node) do |exp, act, msg|
```

Comment:
I liked expected/actual/failure_message better.

Reply:
That's for sure. Corrected 👍🏻